cmd/integration/integration.go

```diff
@@ -855,6 +879,29 @@ func ServeCachedManifestFile() (servingAddress string) {
 }
 
 const (
+	testManifestFile = `kind: Pod
```

Comment:
FYI: I found an easy way to test both modes (i.e. reading both Pods and ContainerManifest from http channel). Basically, in the integration test we have two Kubelets, so I did small changes that the first one is reading Pods from http, and the second is reading ContainerManifest from http.
I hope that you're fine with that.